content/about/feature-stages/index.md

```diff
@@ -74,23 +77,22 @@ Below is our list of existing features and their current phases. This informatio
 | [Kubernetes: Service Credential Distribution](/docs/concepts/security/#pki)   | Stable
 | [VM: Service Credential Distribution](/docs/concepts/security/#pki)         | Beta
 | [Mutual TLS Migration](/docs/tasks/security/mtls-migration)    | Beta
-| [Authentication policy](/docs/concepts/security/#authentication-policies)  | Alpha
+| [Cert management on Ingress Gateway](/docs/tasks/traffic-management/ingress/secure-ingress-sds) | Beta
+| [Authorization (RBAC)](/docs/concepts/security/#authorization)   | Alpha
 | [End User (JWT) Authentication](/docs/concepts/security/#authentication)  | Alpha
 | [OPA Checker](/docs/reference/config/policy-and-telemetry/adapters/opa/)    | Alpha
-| [Authorization (RBAC)](/docs/concepts/security/#authorization)   | Alpha
-| [TCP Authorization (RBAC)](/docs/tasks/security/authz-tcp) | Alpha
-| [Cert management on Ingress Gateway](/docs/tasks/traffic-management/ingress/secure-ingress-sds) | Alpha
 | [Vault integration](/docs/tasks/security/vault-ca) | Alpha
 | [SDS Integration](/docs/tasks/security/auth-sds/) | Alpha
 
+The 'Authorization (RBAC)' runtime is considered Beta, but it is advertised as Alpha since its API is still subject to backward incompatible change.
```

Comment:
@liminw @linsun I'm proposing this as a way of indicating that the impl is more stable, but the API is still subject to change

Reply:
Sounds good to me. Thanks @duderino

Reply:
@duderino Maybe this wording would work better:
```suggestion
The 'Authorization (RBAC)' runtime is considered Beta. However, its API is still subject to a backwards incompatible change.
Due to this, we advertise it as Alpha.
```